unlisted-xtoken-tokens.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
can we remove this + the xtoken-tokens-without-coingecko-data.json file from the commit as they are not necessary?

Reply:
Thumbs up sir